lib/cli/src/Cardano/CLI.hs

```diff
@@ -804,11 +807,12 @@ data WalletCreateArgs = WalletCreateArgs
     , _gap :: AddressPoolGap
     }
 
-cmdWalletCreate
+cmdWalletCreateFromMnemonic
     :: forall t. (DecodeAddress t, EncodeAddress t)
     => Mod CommandFields (IO ())
-cmdWalletCreate = command "create" $ info (helper <*> cmd) $ mempty
-    <> progDesc "Create a new wallet using a sequential address scheme."
+cmdWalletCreateFromMnemonic =
+    command "create from-mnemonic" $ info (helper <*> cmd) $ mempty
```

Comment:
Shouldn't this actually be:

```hs
cmdWalletCreate :: Mod CommandFields (IO ())
cmdWalletCreate = command "create" $ info (helper <*> cmds) $ mempty
    <> progDesc "Create a new wallet."
  where
    cmds = subparser $ mempty
        <> cmdWalletCreateFromMnemonic
        <> cmdWalletCreateFromPublicKey

-- etc ...
```

i.e., having both as sub-commands of a parent command (like it's done for `cmdMnemonic` for instance)

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1421/commits/eb3a59a43df8cccfca3f48912e290fc519c62316